Editor/Data/Type/DefaultObservableListDrawer.cs

```diff
@@ -0,0 +1,109 @@
+﻿namespace Zinnia.Data.Type
+{
+    using UnityEngine;
+    using UnityEditor;
+    using System;
+    using Zinnia.Utility;
+    using Zinnia.Data.Collection.List;
+
+    [CustomPropertyDrawer(typeof(ObservableList), true)]
+    public class DefaultObservableListDrawer : PropertyDrawer
+    {
+        protected const string collectionLabel = "{0} Elements";
+        protected const string collectionElementLabel = "Element {0}";
+        protected const string collectionEmptyLabel = "List is Empty";
+        protected const float footerSpacing = 2f;
+        protected const float objectPickerPadding = 4f;
+        protected readonly GUIStyle buttonStyle = new GUIStyle("RL Footer");
+        protected readonly GUIContent objectPickerIcon = EditorGUIUtility.TrIconContent("In ObjectField");
+        protected readonly GUIContent buttonIconAdd = EditorGUIUtility.TrIconContent("Toolbar Plus", "Add element to list");
+        protected readonly GUIContent buttonIconRemove = EditorGUIUtility.TrIconContent("Toolbar Minus", "Remove last element from list");
+
+        public override void OnGUI(Rect position, SerializedProperty property, GUIContent label)
+        {
+            {
+                label.tooltip = EditorHelper.GetTooltipAttribute(fieldInfo)?.tooltip ?? string.Empty;
+                EditorGUILayout.PropertyField(property, label, true);
+                object propertyObject = fieldInfo.GetValue(property.serializedObject.targetObject);
+                if (propertyObject == null)
+                {
+                    return;
+                }
+
+                buttonStyle.overflow = new RectOffset(2, 0, 2, 0);
+                buttonStyle.fixedHeight = buttonIconAdd.image.height * 1.01f;
+                buttonStyle.fixedWidth = objectPickerIcon.image.width * 1.5f;
+
+                using (new EditorGUI.IndentLevelScope())
+                {
+                    property.isExpanded = EditorGUILayout.Foldout(property.isExpanded, string.Format(collectionLabel, label.text), true);
```

Comment:
```suggestion
                    property.isExpanded = EditorGUILayout.Foldout(property.isExpanded, collectionLabel, true);
```
Because of the above suggestion to change `collectionLabel`.